metals/src/main/scala/scala/meta/internal/troubleshoot/ProblemResolver.scala

```diff
@@ -141,19 +144,25 @@ class ProblemResolver(
   private def findProblem(
       scalaTarget: ScalaTarget
   ): Option[ScalaProblem] = {
+
+    // We don't publish all mtags for unit test
+    // So check only versions that comes from build-info
+    def isSupportedScalaVersion(version: String): Boolean =
+      if (!allSupportedMtagsRequired)
+        ScalaVersions.isSupportedAtReleaseMomentScalaVersion(version)
+      else mtagsResolver.isSupportedScalaVersion(version)
```

Comment:
Doesn't `requireAllSupportedMtags` sound better? Also, could you expand your comment? I'm afraid I don't get it.

Reply:
This doesn't seem to be needed I think? We only run unit tests for 2.12.x

Reply:
I wasn't happy about adding this thing but there are some tests in unit that uses scala 3 sources.
At least [this one](https://github.com/scalameta/metals/blob/main/tests/unit/src/test/scala/tests/NewFileLspSuite.scala#L330)

I can look at it one more time and modify tests to get rid of it.

Reply:
The change here shouldn't cause it to fail, it doesn't actually use the presentation compiler. Why did it start to fail?

We can for sure move it to slow if needed.